Client/qtTeamTalk/preferencesdlg.cpp

```diff
@@ -209,12 +209,7 @@ PreferencesDlg::PreferencesDlg(SoundDevice& devin, SoundDevice& devout, QWidget
     connect(ui.captureformatsBox, QOverload<int>::of(&QComboBox::currentIndexChanged),
             this, &PreferencesDlg::slotVideoResolutionChange);
     connect(ui.vidtestButton, &QAbstractButton::clicked, this, &PreferencesDlg::slotTestVideoFormat);
-    connect(ui.vidrgb32RadioButton, &QAbstractButton::clicked,
-            this, &PreferencesDlg::slotImageFormatChange);
-    connect(ui.vidi420RadioButton, &QAbstractButton::clicked,
-            this, &PreferencesDlg::slotImageFormatChange);
-    connect(ui.vidyuy2RadioButton, &QAbstractButton::clicked,
-            this, &PreferencesDlg::slotImageFormatChange);
+    connect(ui.vidImgFmtBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &PreferencesDlg::slotImageFormatChange);

```

Comment:
PreferencesDlg::slotImageFormatChange takes a boolean as argument. Shouldn't it be an `int` ?

I experience a crash here:
9  PreferencesDlg::slotVideoCaptureDevChange(int) preferencesdlg.cpp 1564 0x10012f0ea 
10 PreferencesDlg::slotImageFormatChange(bool)    preferencesdlg.cpp 1665 0x100130696

`index` in PreferencesDlg::slotVideoCaptureDevChange(int index) is -1.